src/acq.c

```diff
@@ -75,7 +77,7 @@ bool acq_load(u32 count)
   nap_acq_load_wr_enable_blocking();
   nap_timing_strobe(count);
   if (chBSemWaitTimeout(&load_wait_sem, 1000) == RDY_TIMEOUT) {
-    printf("acq: Timeout waiting for sample load!\n");
+    log_warn("acq: Timeout waiting for sample load!\n");
```

Comment:
Yeah, the `log_*` functions work just like `printf` so we can add any info we want. Do you want me to add the timeout length to these messages? Its fairly low level and the timeout isn't configurable by the user.